content/docs/integrating-with-other-libraries.md

```diff
@@ -85,32 +85,32 @@ class Chosen extends React.Component {
 }
 ```
 
-Notice how we wrapped `<select>` in an extra `<div>`. This is necessary because Chosen will append another DOM element right after the `<select>` node we passed to it. However, as far as React is concerned, `<div>` always only has a single child. This is how we ensure that React updates won't conflict with the extra DOM node appended by Chosen. It is important that if you modify the DOM outside of React flow, you must ensure React doesn't have a reason to touch those DOM nodes.
+Fazladan bir `<div>` içinde `<select>`'i nasıl sardığımıza dikkat edin. Bu gereklidir çünkü Chosen, ona geçtiğimiz `<select>` düğümünün hemen arkasına başka bir DOM öğesi ekleyecektir. Ancak, React söz konusu olduğunda, `<div>`'in her zaman yalnızca tek bir alt öğesi vardır. Bu, React güncellemelerinin, Chosen tarafından eklenmis fazladan DOM düğümüyle çakışmamasını sağlama şeklimizdir.  Önemlidir ki, eğer DOM'u React akışının dışında değiştirirseniz, React'in bu DOM düğümlerine dokunmak için bir nedeni olmadığından emin olmalısınız.
 
-Next, we will implement the lifecycle methods. We need to initialize Chosen with the ref to the `<select>` node in `componentDidMount`, and tear it down in `componentWillUnmount`:
+Ardından, yaşam döngüsü yöntemlerini uygulayacağız. `componentDidMount`'de `<select>` düğümüne ref ile Chosen'i baslatmamiz  ve `componentWillUnmount`'da parçalamamız gerek:
 
 ```js{2,3,7}
 componentDidMount() {
   this.$el = $(this.el);
-  this.$el.chosen();
+  this.$el.chosen(); 
 }
 
 componentWillUnmount() {
   this.$el.chosen('destroy');
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/qmqeQx?editors=0010)
+[**CodePen'de Deneyin**](https://codepen.io/gaearon/pen/qmqeQx?editors=0010)
 
-Note that React assigns no special meaning to the `this.el` field. It only works because we have previously assigned this field from a `ref` in the `render()` method:
+Not edin ki, React'in `this.el` alanına özel bir anlam atamaz. O sadece, öncesinde bu alana `render()` yöntemindeki bir `ref`'den atama yapılmış olduğunda çalışır:
```

Comment:
```suggestion
Not edin ki, React'in `this.el` alanına özel bir anlam atamaz. O sadece, öncesinde bu alana `render()` metodundaki bir `ref`'den atama yapılmış olduğunda çalışır:
```

Reply:
It is the right translation into Turkish for the word of method. If you prefer to use "metodundaki" instead of using "yöntemindeki", you can go with your own choice.